packages/DropdownMenu/test/dropdownMenu.spec.js

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { render, configure } from "@testing-library/react";
+import L10n from "@paprika/l10n";
+import DropdownMenu from "../src";
+
+configure({ testIdAttribute: "data-qa-anchor" });
+
+const evt = document.createEvent("HTMLEvents");
+
+const transitionDelay = 150;
+
+function renderComponent(props = {}) {
+  const renderedComponent = render(
+    <L10n>
+      <DropdownMenu
+        renderTrigger={({ isOpen, handleOpenMenu }) => (
+          <DropdownMenu.Trigger isOpen={isOpen} handleOpenMenu={handleOpenMenu} {...props}>
+            Trigger
+          </DropdownMenu.Trigger>
+        )}
+      >
+        <DropdownMenu.Item onClick={() => {}}>Edit</DropdownMenu.Item>
+        <DropdownMenu.Item onClick={() => {}}>Filter</DropdownMenu.Item>
+        <DropdownMenu.Item
+          isDestructive
+          renderConfirmation={handleCloseMenu => {
+            return (
+              <DropdownMenu.Confirmation
+                confirmLabel="Confirm Delete"
+                onConfirm={() => {
+                  handleCloseMenu();
+                }}
+                onCancel={() => {
+                  handleCloseMenu();
+                }}
+                heading="Delete Button?"
+              />
+            );
+          }}
+        >
+          Delete
+        </DropdownMenu.Item>
+      </DropdownMenu>
+    </L10n>
+  );
+
+  return {
+    ...renderedComponent,
+  };
+}
+
+describe("DropdownMenu", () => {
```

Comment:
Could test focus here as well, can be another PR.

One example
"should focus on trigger after dropdown menu is closed"

Reply:
Ok added but i'm having a few issues with this which im investigating

Reply:
I think there are problems related to https://github.com/acl-services/paprika/pull/113#discussion_r302289468